src/components/NodeDetail.tsx

```diff
@@ -1,66 +1,35 @@
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core'
 import range from 'lodash/range'
+import get from 'lodash/get'
 import React from 'react'
 import { observer } from 'mobx-react'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faCaretLeft, faCaretRight } from '@fortawesome/free-solid-svg-icons'
 import { useTheme } from 'emotion-theming'
 import { Theme } from '@emotion/types'
-import { NodeUserInput } from './NodeUserInput'
 import { styles } from './NodeDetail.style'
+import { NodeUserInput } from './NodeUserInput'
+import CollapsibleSection from './CollapsibleSection'
+import Term from './Term'
+import Media from './Media'
 import { getMedia } from '../firestore/firestore'
 
+const SourceLink = (link) => (
+  <a href={link} target="_blank">
```

Comment:
I think link here is an object with a `link` key in it. So we need to pass in item.link.link on line 73? This doesn't make sense to me but it seems to be what we have coming from the back-end.